docs/pages/versions/unversioned/sdk/filesystem.md

```diff
@@ -284,6 +328,44 @@ Returns a Promise that resolves to an object with the following fields:
 
 - **md5 (_string_)** -- Present if the `md5` option was truthy. Contains the MD5 hash of the file.
 
+### `FileSystem.uploadAsync(url, fileUri, options)`
+
+Upload the contents of the file pointed by `fileUri` to the remote url.
+
+#### Arguments
+
+- **url (_string_)** -- The remote URL, where the file will be sent.
+
+- **fileUri (_string_)** -- The local URI of the file to send. The file must exist.
+
+- **options (_object_)** -- A map of options:
+
+  - **headers (_object_)** -- An object containing all the HTTP header fields and their values for the download network request. The keys and values of the object are the header names and values respectively.
+
+  - **httpMethod (_String_)** -- The request method. Accepts values: 'POST', 'PUT', 'PATCH. Default to 'POST'.
+
+  - **sessionType (_FileSystemSessionType_)** -- (iOS only) A session type. Determines if tasks can be handled in the background. On Android, sessions always work in the background and you can't change it. Defaults to `FileSystemSessionType.BACKGROUND`.
+
+  - **uploadType (_FileSystemUploadOptions_)** -- Upload type determines how the file will be sent to the server. Default to `FileSystemUploadType.BINARY_CONTENT`.
+
+  If `uploadType` is equal `FileSystemUploadType.MULTIPART`, more options are available:
+
+  - **fieldName (_string_)** -- The name of the field which will hold uploaded file. Defaults to the file name without an extension.
+
+  - **mimeType (_string_)** -- The MIME type of the provided file. If not provided, the module will try to guess it based on the extension.
+
+  - **parameters (_Record<string, string>_)** -- Additional HTTP parameters.
```

Comment:
By additional HTTP parameters you mean other form properties? Like… for `fieldName: "image"` under `"image"` there would be an image and here I could pass `{ name: "Stanley" }` to have `"Stanley"` under `"name"`?

Reply:
Exactly ;)